tests/resources/spec.json

```diff
@@ -593,29 +603,86 @@
         ]
       }
     },
+    "/mock/tests/general/400-error": {
+      "get": {
+        "summary": "Mock a an 400 failed request",
+        "operationId": "generalError400",
+        "consumes": ["application/json"],
+        "produces": [],
+        "tags": ["general"],
+        "description": "",
+        "responses": {
+          "500": { "description": "File", "schema": { "type": "file" } }
+        },
+        "x-appwrite": {
+          "method": "error400",
+          "weight": 185,
+          "cookies": false,
+          "type": "",
+          "demo": "general/error400.md",
+          "edit": "https://github.com/appwrite/appwrite/edit/masterMock an 400 error",
+          "rate-limit": 0,
+          "rate-time": 3600,
+          "rate-key": "url:{url},ip:{ip}",
+          "scope": "public",
+          "platforms": ["client", "server"],
+          "packaging": false
+        },
+        "security": [{ "Project": [] }]
+      }
+    },
+    "/mock/tests/general/500-error": {
+      "get": {
+        "summary": "Mock a an 500 failed request",
+        "operationId": "generalError500",
+        "consumes": ["application/json"],
+        "produces": [],
+        "tags": ["general"],
+        "description": "",
+        "responses": {
+          "500": { "description": "File", "schema": { "type": "file" } }
+        },
+        "x-appwrite": {
+          "method": "error500",
+          "weight": 186,
+          "cookies": false,
+          "type": "",
+          "demo": "general/error500.md",
+          "edit": "https://github.com/appwrite/appwrite/edit/masterMock an 500 error",
+          "rate-limit": 0,
+          "rate-time": 3600,
+          "rate-key": "url:{url},ip:{ip}",
+          "scope": "public",
+          "platforms": ["client", "server"],
+          "packaging": false
+        },
+        "security": [{ "Project": [] }]
+      }
+    },
     "/mock/tests/general/empty": {
       "get": {
         "summary": "Mock a post request for SDK tests",
         "operationId": "generalEmpty",
         "consumes": ["application/json"],
-        "produces": ["application/json"],
+        "produces": [],
```

Comment:
Not sure why this was changed everywhere? What's the significance?

Reply:
This is used mainly in the docs, because of a bug it was missing the previous time I generated the spec